packages/lib/src/dimension-mapper/SlicingSlider.module.css

```diff
@@ -29,38 +32,34 @@
 
 .track {
   flex: 1;
-  margin: calc(var(--thumb-height) / 2) 0;
-  border-left: 2px solid var(--secondary-dark);
-  border-radius: 2px;
+  margin: calc(var(--thumb-size) / 2) 0;
+  border-left: var(--track-width) solid var(--track-color);
+  border-radius: var(--track-width);
 }
 
 .thumb {
   display: flex;
   align-items: center;
   justify-content: center;
-  padding: 0.125rem;
-  background-color: var(--primary-bg);
-  border: 2px solid var(--secondary);
-  border-radius: 1.5em;
-  height: var(--thumb-height);
-  min-width: 1.5rem;
-  font-size: 0.625rem;
+  padding: 0.125rem 0.25rem;
```

Comment:
Increasing horizontal padding inside the thumb a tiny bit.

Reply:
And therefore triggering a cascade of new screenshots :joy_cat: